.github/workflows/e2e-tests.yml

```
@@ -114,7 +114,7 @@ jobs:
         mv version.properties resources/
 
     - name: Run Cypress tests on ${{ matrix.folder }}
-      run: yarn run test-cypress-no-build --folder ${{ matrix.folder }} --record --key ${{ secrets.CURRENTS_KEY }} --group ${{matrix.folder}} --ci-build-id "${{ github.run_id }}-${{ github.run_attempt }}"
+      run: yarn run test-cypress-no-build --folder ${{ matrix.folder }} --record --key ${{ secrets.CURRENTS_KEY }} --group ${{matrix.edition}} --ci-build-id "${{ github.run_id }}-${{ github.run_attempt }}"
```

I'm sorry but this doesn't make sense.
With this proposal we'd have only two groups: `oss` and `ee` for all tests. The point was to **add** edition to the existing group name.

```suggestion
      run: yarn run test-cypress-no-build --folder ${{ matrix.folder }} --record --key ${{ secrets.CURRENTS_KEY }} --group ${{ matrix.folder }}-${{matrix.edition}} --ci-build-id "${{ github.run_id }}-${{ github.run_attempt }}"
```